docs/organizing-and-exploring-results-in-the-ui/index.rst

```diff
@@ -3,14 +3,36 @@ Working with Neptune UI
 
 |Youtube Video dashboard|
 
-**More info:**
+Neptune UI has 3 major sections: Experiments, Notebooks and Wiki.
 
-.. toctree::
-   :maxdepth: 2
+**Experiments** section is the place where you can:
+
+- organize experiments into groups and views
+- filter and search experiments
+- compare experiments
+- drill down to experiment details like charts and code
+
+**Notebooks** section is where you can:
+
+- explore notebooks and notebook checkpoints
+- compare notebook checkpoints
 
-   Experiment dashboard <experiment-dashboard/index.rst>
-   Notebooks <notebooks/index.rst>
-   Wiki <wiki/index.rst>
+In the **Wiki** section you can:
+
+- add notes about experiments and link to them
+- discuss experiments with your team
+
+**For more info go to one of the pages:**
+
+.. toctree::
+   :maxdepth: 1
```

Comment:
Maybe this should mirror the bullets from above? So you will know immediately where to go for more info.